crates/bevy_ui/src/update.rs

```diff
@@ -37,7 +42,12 @@ pub fn update_clipping_system(
 fn update_clipping(
     commands: &mut Commands,
     ui_children: &UiChildren,
-    node_query: &mut Query<(&Node, &GlobalTransform, &Style, Option<&mut CalculatedClip>)>,
+    node_query: &mut Query<(
+        &ComputedNode,
+        &GlobalTransform,
+        &Node,
+        Option<&mut CalculatedClip>,
+    )>,
     entity: Entity,
     mut maybe_inherited_clip: Option<Rect>,
 ) {
```

Comment:
There is a variable still called `style` just below this comment, I can't comment on the line itself unfortunately